src/braket/circuits/gates.py

```diff
@@ -2610,12 +2919,26 @@ def gpi2(
 
 
 class MS(TripleAngledGate):
-    """IonQ Mølmer-Sørenson gate.
+    r"""IonQ Mølmer-Sørensen gate.
+
+        Unitary matrix:
```

Comment:
nit, to make this indentation consistent with the rest of the gates:
```suggestion
    Unitary matrix:
```